packages/react/src/components/TextInput/TextInput.js

```diff
@@ -72,7 +72,7 @@ const TextInput = React.forwardRef(function TextInput(
     type,
     ref,
     className: textInputClasses,
-    title: placeholder,
```

Comment:
I _think_ this is considered a breaking change since existing code is now incorrect so might be worth flagging this